js/components/adPanel.js

```diff
@@ -57,7 +57,12 @@ var AdPanel = React.createClass({
     var adTopBarItems = [];
 
     // // Ad title
-    var adTitle = this.props.currentAdsInfo.currentAdItem.name;
+    var adTitle = "Unknown";
+    if (this.props.currentAdsInfo.currentAdItem.ooyalaAds && this.props.contentTree.title) {
```

Comment:
Can we add a null check for both this.props.currentAdsInfo and this.props.currentAdsInfo.currentAdItem and also this.props.contentTree?